test/test_timesteparrays.jl

```diff
@@ -611,6 +611,23 @@ y_mat = TimestepMatrix{VariableTimestep{y_years}, Int, 1}(time_dim_val[:,:,2])
 
 @test eltype(x_vec) == eltype(y_vec) == eltype(y_vec) == eltype(y_mat) == eltype(time_dim_val)
 
+# begin syntax is depreacated v1.0.0 - v1.3.0 
+if(VERSION > v"1.3.0")
```

Comment:
@ckingdon95 This doesn't seem to be helping, might be because of how the file is parsed, but `v1.2.0` and `v1.3.0` catch an error in this block. I'll look into it.